.gitignore

```diff
@@ -69,3 +69,6 @@ media/
 
 #Pycharm projects
 .idea/
+
+#VSCode Sphinx Extension
+docs/.vscode/settings.json
```

Comment:
I think it does not need to be included. Not everyone uses vscode. Keep this settings in your local .gitignore without push to github repository.

Reply:
Ok thanks for pointing that out!
I have made the necessary changes and pushed again. Please review it.
Thanks